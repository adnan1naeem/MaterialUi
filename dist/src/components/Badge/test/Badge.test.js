"use strict";
var __importStar = (this && this.__importStar) || function (mod) {
    if (mod && mod.__esModule) return mod;
    var result = {};
    if (mod != null) for (var k in mod) if (Object.hasOwnProperty.call(mod, k)) result[k] = mod[k];
    result["default"] = mod;
    return result;
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const React = __importStar(require("react"));
const __1 = require("../");
const react_test_renderer_1 = __importDefault(require("react-test-renderer"));
test('Badge component with required fields', () => {
    const badge = react_test_renderer_1.default.create(React.createElement(__1.Badge, { badgeContent: '123' }, "Hello")).toJSON();
    expect(badge.props.style).toEqual(undefined);
    expect(badge).toMatchSnapshot();
});
test('Badge component with style', () => {
    const badge = react_test_renderer_1.default.create(React.createElement(__1.Badge, { badgeContent: '123', style: { 'background-color': 'green' } }, "hello")).toJSON();
    expect(badge.props.style).toEqual({ 'background-color': 'green' });
    expect(badge).toMatchSnapshot();
});
//# sourceMappingURL=Badge.test.js.map