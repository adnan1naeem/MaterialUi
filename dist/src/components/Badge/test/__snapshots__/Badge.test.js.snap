// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge component with required fields 1`] = `
<span
  className="MuiBadge-root-1"
>
  Hello
  <span
    className="MuiBadge-badge-2"
  >
    123
  </span>
</span>
`;

exports[`Badge component with style 1`] = `
<span
  className="MuiBadge-root-1"
  style={
    Object {
      "background-color": "green",
    }
  }
>
  hello
  <span
    className="MuiBadge-badge-2"
  >
    123
  </span>
</span>
`;
